modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/index.tsx

```diff
@@ -15,22 +15,62 @@
 import ReactDOM from 'react-dom';
 
 import ClayIconProvider from './common/context/ClayIconProvider';
-import AgentPortal from './routes/agent-portal/pages/AgentPortal';
+import Dashboard from './routes/dashboard/pages/Dashboard';
 
 import './common/styles/index.scss';
+import Applications from './routes/applications/pages/Applications';
+import Claims from './routes/claims/pages/Claims';
+import Policies from './routes/policies/pages/Policies';
+import Reports from './routes/reports/pages/Reports';
+
+interface Props {
```

Comment:
@silasbsi I would like to recommend to use `type` instead of interface` whenever possible

```suggestion
type Props {
```

Reply:
https://www.educba.com/typescript-type-vs-interface/